1.9/usage/pods/quick-start.md

```diff
@@ -0,0 +1,31 @@
+---
+post_title: Quick Start
+menu_order: 0
+---
+
+# Prerequisites: A DC/OS Cluster, DC/OS CLI installed
+
+# Launch a Pod from the DC/OS CLI
+
+1. Create a JSON application definition with the following code. In this example, we will call the file `simple-pod.json`.
+
+```json
+{
+  "id": "/simplepod",
+  "scaling": { "kind": "fixed", "instances": 1 },
+  "containers": [
+    {
+      "name": "sleep1",
+      "exec": { "command": { "shell": "sleep 1000" } },
+      "resources": { "cpus": 0.1, "mem": 32 }
+    }
+  ],
+  "networks": [ { "mode": "host" }, "name": "my-virtual-network-name" ]
+}
+```
+
+1. Launch the pod on DC/OS with the following DC/OS CLI command: `dcos marathon pod add simple-pod.json`.
```

Comment:
This command should probably have the three backticks rather than single to call it out similar to the other.